JunkyardLoad/Properties/ServiceDependencies/dd-aas-junkyard-load-v4/storage1.arm.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2018-05-01/subscriptionDeploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceGroupName": {
+      "type": "string",
+      "defaultValue": "apm-aas-junkyard",
```

Comment:
shouldn't it be `dd-aas-junkyard-load-v4` ?

Reply:
_sigh_ I have no idea, this is what VS generated 🤷‍♂️